obal/data/playbooks/library/rpmspec_changelog_check.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
```

Comment:
Using `#!/usr/bin/env python` is more common. That way it also works in a virtualenv where the python is in a different location.